library/std/src/sync/rwlock.rs

```diff
@@ -136,6 +137,21 @@ impl<T> RwLock<T> {
             data: UnsafeCell::new(t),
         }
     }
+
+    /// Creates a new instance of an `RwLock<T>` which is unlocked, and wraps it in an [`Arc`].
+    ///
+    /// [`Arc`]: ../../std/sync/struct.Arc.html
```

Comment:
```suggestion
    ///
    /// [`Arc`]: ../../std/sync/struct.Arc.html
```

Reply:
Yes, since `Arc` is in scope this will be resolved by intra-doc links.